Errors in passwordsubset.cpp build

Platform: FreeBSD 11.1
Compiler: clang 4.0
wxwidgets: 3.0.3.1 GTK2
Both: 1.02.1BETA, 1.03BETA and master(9df771c) builds 

```
clang++ -c -o GCCUnicodeRelease/pwsqrcodedlg.o -O2 -fno-strict-aliasing -pipe -march=native -fstack-protector  --stdlib=libc++ -I/usr/local
/include `/usr/local/bin/wxgtk2u-3.0-config --debug=no --unicode=yes --inplace --cxxflags` -DUNICODE -DNDEBUG -I../.. -fPIC  -O -Wall -DUSE
_XML_LIBRARY=XERCES -DWCHAR_INCOMPATIBLE_XMLCH -DNO_YUBI   -std=c++11 pwsqrcodedlg.cpp
clang++ -c -o GCCUnicodeRelease/passwordsubset.o -O2 -fno-strict-aliasing -pipe -march=native -fstack-protector  --stdlib=libc++ -I/usr/loc
al/include `/usr/local/bin/wxgtk2u-3.0-config --debug=no --unicode=yes --inplace --cxxflags` -DUNICODE -DNDEBUG -I../.. -fPIC  -O -Wall -DU
SE_XML_LIBRARY=XERCES -DWCHAR_INCOMPATIBLE_XMLCH -DNO_YUBI   -std=c++11 passwordsubset.cpp
passwordsubset.cpp:243:27: error: no matching constructor for initialization of 'std::wistringstream' (aka 'basic_istringstream<wchar_t>')
      std::wistringstream is(pos_str.wc_str());
                          ^  ~~~~~~~~~~~~~~~~
passwordsubset.cpp:283:42: error: no viable conversion from 'const wxScopedWCharBuffer' (aka 'const wxScopedCharTypeBuffer<wchar_t>') to
      'const StringX' (aka 'const basic_string<wchar_t, std::char_traits<wchar_t>, S_Alloc::SecureAlloc<wchar_t> >')
    PWSclipboard::GetInstance()->SetData(reduced_str.wc_str());
                                         ^~~~~~~~~~~~~~~~~~~~
```

--- a/src/ui/wxWidgets/passwordsubset.cpp
+++ b/src/ui/wxWidgets/passwordsubset.cpp
@@ -240,7 +240,7 @@ void CPasswordSubset::OnChar( wxKeyEvent& event )
       m_vals->Clear();
       m_error->SetLabel(wxEmptyString);
       
-      std::wistringstream is(pos_str.wc_str());
+      std::wistringstream is(static_cast<const wchar_t*>pos_str.wc_str());
       int pos;
       while (is >> pos) {
 	if (pos > 0 && pos <= N)
@@ -280,7 +280,7 @@ void CPasswordSubset::OnBitmapbuttonClick( wxCommandEvent& event )
     wxASSERT(val_str.length() % 2 == 0);
     for (size_t i = 0; i < val_str.length(); i += 2)
       reduced_str += val_str[i];
-    PWSclipboard::GetInstance()->SetData(reduced_str.wc_str());
+    PWSclipboard::GetInstance()->SetData(static_cast<const wchar_t*>reduced_str.wc_str());
   }
 }
 

